lib/html_formatter.js

```diff
@@ -80,6 +80,7 @@ module.exports = (function () {
             data: {
                 status: step.result ? step.result.status : '',
                 errorDetails: step.result ? toHtmlEntities(step.result.error_message) : '',
+                duration: step.result.duration ? (step.result.duration / 1000000000).toFixed(2) + 's' : 'no data',
```

Comment:
no wlasnie tak myslalem, ale w koncu gdzies znalazlem ze nie.
tak wyglada w jsonie: 1781558970
więc to nie są milisekundy bo musialoby byc np 2000 = 2sec
nie za bardzo mialem jak to sprawdzic wiec mierzylem stoperem :), sumowalem tez recznie duration i sprawdzalem czy sie mniej wiecej zgadza (bo nie wiem dokladnie od jakiego miejsca duration jest mierzone i wypluwane do jsona).
Masz jakis pomysl jak to sprawdzic?